### PR TITLE
Remove starting Crossvec from SCP-035 on spawn

### PR DESCRIPTION
**Pull request by Darkita162**
Seriously, whos idea was this to give him the best weapon in the game to start of with?

### CODE REVIEW

File: addons/sourcemod/configs/scp_sf/classes.cfg [hunk 1]
```diff
@@ -737,15 +737,12 @@
 
 		"items"
 		{
-			"1"	"203"	// Crossvec
+			"1"	"30758"	// Prinny Machete #This guy shouldn't spawn with a gun right at the start. If he can find a gun, good for him, just don't give him one of the best in the mode to fuck people over with.
```
Comment (Mikusch) [inline addons/sourcemod/configs/scp_sf/classes.cfg:740]: Please leave your opinion out of code comments.
```suggestion
			"1"	"30758"	// Prinny Machete
```

File: addons/sourcemod/configs/scp_sf/classes.cfg [hunk 2]
```diff
@@ -737,15 +737,12 @@
...
 			"2"	"30003"	// Keycard
 			"3"	"30020"	// Combat Armor
 			"4"	"5"	// None
 		}
-		"ammo"
-		{
-			"2"	"80"	// 9mm
-		}
+
```
Comment (Mikusch) [inline addons/sourcemod/configs/scp_sf/classes.cfg:745]: Try not to change the indentation/whitespace of the file.

**Review by Mikusch — CHANGES_REQUESTED** [review]
The Crossvec for SCP-035 is also defined in map config files, please go through them all to make sure you're not missing one.

**Comment by FortyTwoFortyTwo** [comment]
`scp_turtle_support.cfg` map config also needs to be updated as it have scp035 with Crossvec

**Pull request closed without merging**